solr/core/src/test/org/apache/solr/cli/StreamToolTest.java

```diff
@@ -185,11 +183,11 @@ public void testReadStream() throws Exception {
   @Test
   @SuppressWarnings({"unchecked", "rawtypes"})
   public void testLocalCatStream() throws Exception {
-    File localFile = File.createTempFile("topLevel1", ".txt");
-    populateFileWithData(localFile.toPath());
+    Path localFile = Files.createTempFile("topLevel1", ".txt");
+    populateFileWithData(localFile);
 
     StreamTool.LocalCatStream catStream =
-        new StreamTool.LocalCatStream(localFile.getAbsolutePath(), -1);
+        new StreamTool.LocalCatStream(localFile.toAbsolutePath().toString(), -1);
```

Comment:
Should we just pass in a path here?

Reply:
agreed to leave CatStream be.  I *could* see adding a Path convenience constructor there but it's out of scope of this PR IMO.